html/docs/files.html

```diff
@@ -95,6 +95,19 @@ <h4>Setup</h4>
               only affects your ability to <code>undo</code> changes.
             </p>
 
+						<p> 
+							The <code>tags.data</code> contains information about how often a
+							certain tag is used to make Timewarrior able to notify the user
```

Comment:
Instead of "to make Timewarrior able to" I would prefer "to enable Timewarrior to"